src/uu/tar/src/operations/create.rs

```diff
@@ -110,3 +130,15 @@ fn get_tree(path: &Path) -> Result<Vec<PathBuf>, std::io::Error> {
 
     Ok(paths)
 }
+
+fn normalize_path(path: &Path) -> Option<PathBuf> {
```

Comment:
we have it in uucore, please use it:
https://docs.rs/uucore/latest/uucore/fs/fn.normalize_path.html